alta-back-end/src/main/java/com/alta/service/impl/DefaultStudentService.java

```diff
@@ -39,10 +39,8 @@ public List<StudentDto> findAll() {
     }
 
     @Override
-    public List<StudentDto> findAllByIds(List<Integer> studentIds) {
-        return studentRepository.findAllById(studentIds).stream()
-                .map(studentMapper::toStudentDto)
-                .toList();
+    public List<Student> findAllByIds(List<Integer> studentIds) {
+        return studentRepository.findAllById(studentIds);
```

Comment:
Why you change dto to entity?

Reply:
I need to get entity from that method, look where it is called